src/biokbase/narrative/tests/test_appmanager.py

```diff
@@ -24,21 +23,41 @@
 
 SEMANTIC_VER_ERROR = "Semantic versions only apply to released app modules."
 TOKEN_ID = "ABCDE12345"
+UNKNOWN_APP_ID = "Unknown app id"
+READS_FILE_1 = "reads file 1"
+READS_FILE_2 = "reads file 2"
+NEW_READS_SET = "New Reads Set"
+READS_OBJ_1 = "rhodobacterium.art.q20.int.PE.reads"
+READS_OBJ_2 = "rhodobacterium.art.q10.PE.reads"
 
 
-def mock_agent_token(*args, **kwargs):
-    return dict({"user": "testuser", "id": TOKEN_ID, "token": "abcde"})
+JOB_COMM_MOCK = "biokbase.narrative.jobs.appmanager.JobComm"
+GET_AGENT_TOKEN = "biokbase.narrative.jobs.appmanager.auth.get_agent_token"
+CLIENTS_AM = "biokbase.narrative.jobs.appmanager.clients.get"
+CLIENTS_AM_SM = "biokbase.narrative.jobs.appmanager.specmanager.clients.get"
+CLIENTS_SM = "biokbase.narrative.jobs.specmanager.clients.get"
+CLIENTS = "biokbase.narrative.clients.get"
+
+
+with mock.patch(CLIENTS, get_mock_client):
+    sm = SpecManager()
+    sm.reload()
+    TEST_SPECS = copy.deepcopy(sm.app_specs)
```

Comment:
all spec data referenced in this (and other) tests now comes from the `app_specs.json` file => no need for a live connection